eagle-core/eagle-alert-parent/eagle-alert/alert-common/src/main/java/org/apache/eagle/alert/engine/model/AlertStreamEvent.java

```diff
@@ -35,6 +35,7 @@
     private StreamDefinition schema;
     private String createdBy;
     private long createdTime;
+    private boolean isDuplicate = false;
```

Comment:
Looks this is only for test verification. Could we avoid add non-required method in stream event, instead use check the statistic info in deduplicator for test verify?